src/core/connection.c

```diff
@@ -2003,6 +2029,38 @@ QuicConnStart(
         goto Exit;
     }
 
+    if (Connection->Settings.IsSet.EncryptionOffloadEnabled) {
+        CXPLAT_QEO_CONNECTION Offloads[] = {
```

Comment:
You need to fill out the structs entirely (CIDs and key info), which means you can only fill these out when all the info is available, which is after the handshake completes and the keys are available, see `QuicCryptoProcessTlsCompletion` and maybe `QuicPathSetActive` too.